docs/tech_primer.md

```diff
@@ -0,0 +1,157 @@
+
+How data is moved around on-prem
+
+-   **Passive**: The device has only passive components (thermistor, dry contact)
+-   **Analog**: The device uses an analog connection (1-10V, 4-20mA etc)
+-   **Serial**: The device does not have an IP address, rather uses a wired serial connection.
+-   **Networked**: The device has its own IP address and presence on the wired or wireless IP network.
+-   **Bridge connected**: The device is on an isolated wired or wireless network behind a gateway device that is networked. 
+
+### Ingestion
+
+How data gets up into the cloud
+
+-   **Direct**: Maintains an authenticated connection directly to Cloud IoT Core.
+    -   Must be networked
+    -   Has a unique private auth key
+-   **Gateway**: A direct device that manages data for other proxied devices.
+    -   Bind & attach of proxied devices
+    -   Encapsulation anti-pattern
+-   **Proxied**: A device that is not direct and is managed through a gateway.
+    -   Proxied networked devices, e.g. BACnet/IP
+    -   Proxied non-networked devices, e.g. RS485, DALI
+-  **Cloud-Cloud**: A device is managed by a third party who provides a cloud connector
+    -   Approved third party managed service
+    -   UDMI virtual device server connects to third party cloud API
+
+### Representation
+
+How data is semantically represented
+
+-   **Reporting**: Has an entry in Cloud IoT Core and reports telemetry data.
+    -   Reporting connected devices
+    -   Reporting proxied devices
+-   **Logical**: Has a semantic representation in the site building config.
+-   **Virtual**: Logical-but-not-reporting.
+
+## Information Sources
+
+-   **Qualification Summary** - One for each type of networked device
+-   **Digital Building Device Register** - Comprehensive list of all devices on a project
+-   **Site Model** - Any ingestion device (direct, gateway, or proxy)
+-   **Building Config** - Does omit networked devices that don't report
+
+## Cloud IoT Core Definitions
+
+-   Project
+    -   Administrative domain (user auth and billing)
+    -   Can contain multiple registries (buildings)
+-   Registry
+    -   Not the same as the Digital Building Device Register
+    -   One Registry per building site (one site_model, one building config)
+-   Device
+    -   May be direct or proxied
+    -   Maps to a reporting device
+-   Gateway
+    -   Proxies for non-authenticating devices
+    -   Represented in Site Model, but maybe not Building Config
+
+
+# Base Tools & Capabilities
+
+## Device Qualification 
+
+[Device qualification](https://github.com/faucetsdn/daq/blob/master/docs/qualification.md)
+qualifies device types as meeting a baseline connectivity and security
+requirement by performing a series of predefined tests against a device.
+Qualification can be automated using the [DAQ tool](https://github.com/faucetsdn/daq/). 
+The output of DAQ is a report. 
+[example report](https://github.com/faucetsdn/daq/blob/master/docs/device_report.md))
+
+### Required Actions
```

Comment:
The wording of this is problematic since it's getting into "process" rather than technical considerations. This needs to be stated in the form of "what needs to exist for this to proceed" and then "what needs to happen to know that it is completed"